parts/kubernetesmastercustomdata.yml

```diff
@@ -298,68 +296,28 @@ write_files:
   content: !!binary |
     MASTER_ADDON_HEAPSTER_DEPLOYMENT_B64_GZIP_STR
 
-- path: /etc/kubernetes/addons/default-storage-class.yaml
-  permissions: "0644"
-  encoding: gzip
-  owner: "root"
-  content: !!binary |
-    MASTER_ADDON_DEFAULT_STORAGE_CLASS_B64_GZIP_STR
-
-- path: "/etc/systemd/system/kubelet.service"
+- path: "/etc/default/kubelet"
   permissions: "0644"
   owner: "root"
   content: |
-    [Unit]
-    Description=Kubelet
-    Requires=docker.service
-    After=docker.service
-
-    [Service]
-    Restart=always
-    ExecStartPre=/bin/mkdir -p /var/lib/kubelet
+    KUBELET_CLUSTER_DNS={{{kubeDnsServiceIP}}}
+    KUBELET_API_SERVERS=<<<concat('https://', variables('masterPrivateIpAddrs')[copyIndex()], ':443')>>>
+    KUBELET_IMAGE={{{kubernetesHyperkubeSpec}}}
+    KUBELET_NETWORK_PLUGIN=
```

Comment:
What implications does this have for pods scheduled on master? Will they be able to start if they are not on the host network?